ears.c

```
@@ -0,0 +1,110 @@
+#include "Ears.h"
+
+void init_audio_data(AudioData **audio_data) {
+    if (!*audio_data) {
```

Cppcheck (2.10-2)

[warning] Uninitialized variable

(uninitvar)